src/Http/Controllers/Controller.php

```diff
@@ -53,9 +53,11 @@ public function insertUpdateData($request, $slug, $rows, $data)
             $content = $this->getContentBasedOnType($request, $slug, $row);
 
             // Set the content back to the previous value when there is really now input for this field
-            if (is_null($content) && is_null($request->input($row->field))) {
-                if (isset($data->{$row->field})) {
-                    $content = $data->{$row->field};
+            if (is_null($content)) {
+                if (is_null($request->input($row->field))) {
+                    if (isset($data->{$row->field})) {
```

Comment:
I will change the code this time since I don't want to get into a discussion about coding style. I have enough stress with my colleague in my day-job for such discussion already.

But: (no offense, just my opinion)
why not when it improves the readability.  We have a php_cs to watch for the coding style and there is no complaint about that. Furthermore I would love to see more code comments . I've had a hard time to understand some part of the code. Next time you are warmly welcome to change my pull request directly if you want to change anything in coding style. I'm not mocking you here, @marktopper did that already in one of my pull request and I'm totally fine with that :) . Cheers.